### DISCUSSION TITLE
ROCm support

### DISCUSSION THREAD

**Discussion by TanYingHao** [pyg-team/pytorch_geometric#4200] [Q&A]
Hi can I check if there is ROCm support for pytorch_geometric? Does this library uses custom kernels for CUDA?

**Comment by rusty1s** [comment]
Currently, we rely on custom CUDA code for PyG, but are in the process of making most of this directly available inside PyTorch core. As such, we do not support ROCm at the moment, but might be able to do so in the future.

**Reply by MelihDarcanxyz** [reply]
I'm 3 years late, the landscape changed a lot, so again I wanted to ask. Since pytorch officially supports ROCm now, are there any plans to provide ROCm support or is it already here?

From my understanding, it's possible to use vanilla pytorch without any change, by just installing the ROCm one. From your comment, I understand you used to leverage custom CUDA kernels 3 years ago, is this still the case?

Thanks a lot for your time, and effort!
